src/flight_booking_agent/agents/utils.py

```
@@ -0,0 +1,17 @@
+AIRPORT_MAP = {
+    'hà nội': 'HAN', 'hanoi': 'HAN', 'nội bài': 'HAN',
+    'hồ chí minh': 'SGN', 'ho chi minh': 'SGN', 'hcm': 'SGN',
+    'sài gòn': 'SGN', 'sai gon': 'SGN', 'tân sơn nhất': 'SGN',
+    'đà nẵng': 'DAD', 'da nang': 'DAD',
+    # Thêm các sân bay khác...
+}
+
+def get_iata_code(location_name: str) -> str | None:
+    if not location_name: return None
```

Ruff (0.13.2)

Multiple statements on one line (colon)

(E701)